py/tests/integration/test_use_cases.py

```diff
@@ -170,16 +170,16 @@ def test_node_discovery():
     carol_api = common.external_api(carol_node)
     carol_top = carol_api.get_top()
     gen_hash = common.genesis_hash(carol_api)
-    ping_obj = Ping(source="localhost",
+    ping_obj = Ping(source="http://localhost:1234",
```

Comment:
A ping with a `source` with an invalid url is considered to be blocked by default.